src/Utils.ts

```diff
@@ -35,3 +36,40 @@ export function parseIntoDictionary(arrOfObjects: ParseDict[], lhs: string,
   }
   return outDict;
 }
+
+export function touchstoneFriendly(touchstone: string[] | undefined): string {
```

Comment:
This may be more defensive than it needs to be - touchstone names are generally quite reliable. But I thought it would be preferable to show suboptimal header text than to crash out here.